x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_overview_header/engine_overview_header.tsx

```diff
@@ -19,6 +20,13 @@ export const EngineOverviewHeader: React.FC<> = () => {
   if (enterpriseSearchUrl) {
     buttonProps.href = `${enterpriseSearchUrl}/as`;
     buttonProps.target = '_blank';
+    buttonProps.onClick = () =>
```

Comment:
Just wondering out loud if this will execute, or if the page will navigate away before this can execute?

Reply:
These are set on all `target="_blank"` links only, so it'll always work because the page itself isn't navigating away to anything.

I'm aware there's nothing stopping another dev from using the sendTelemetry helper on a non target blank link, but I don't think it's worth trying to enforce that right now because eventually (when all of AS is on Kibana) we won't have any links opening to a new AS tab in any case.

Reply:
Makes send to me.

Reply:
I can't even type